### DISCUSSION TITLE
docker error when flashing?

### DISCUSSION THREAD

**Discussion by cli48** [bitcraze/discussions#1271] [Q&A] [answered]
![image](https://github.com/bitcraze/discussions/assets/121372903/bdf9ab6d-92d6-4ddc-a2b9-9af0308c2018)

Hello,

I am currently running virtual box with an ubuntu focal image (20.04.6 LTS).
I'm having an issue attempting to utilize the JTAG olimex to program the AI deck 1.1. Says "docker: unknown server OS: "

What I've done so far, 
1. Checked docker images and ensured the image was commited as 'aideck-with-autotiler'
2. In correct directory
3. Checked docker daemon response (is alive)

Please let me know what I am doing wrong. I used these links as reference.
https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/simple-examples/wifi-streamer/ 
https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/infrastructure/flashing/

Command I am attempting to run:
docker run --rm -v ${PWD}:/module --device /dev/ttyUSB0 --privileged -P bitcraze/aideck tools/build/make-example examples/other/wifi-img-streamer flash

**Comment by gemenerik** [comment, marked as answer]
Did you follow the [post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to run docker without root?
